static/panes/editor.ts

```diff
@@ -870,7 +867,7 @@ export class Editor extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Edit
     }
 
     changeLanguage(newLang: string): void {
-        if (newLang === 'cmake' && languages.cmake) {
+        if (newLang === 'cmake') {
```

Comment:
Not confident in this change, TS thinks `languages.cmake` is always truthy

Reply:
oh hmm @partouf maybe can recall why? Perhaps at one point `languages.cmake` could be undefined, or could be now but is incorrectly typed?